### DISCUSSION TITLE
Option to center Command/Search Palettes relative to Nota's window

### DISCUSSION THREAD

**Discussion by vladstudio** [notaapp/talk#269] [Feature Requests]
I imagine this is totally subjective, but I rais my eyebrows every time I open Nota's command palette, and it appears on bottom-right corner. Basically every other app - Notion, VSCode, Obsidian, Vivaldi, Alfred, Raycast, you name it - show these kinds of popups centered, often with an overlay. I'd argue this becomes a common UI pattern.

Suggestion - add a "Command Palette position" preference, with options "Centered", "Bottom Right".

**Comment by astoilkov** [comment]
Yeah, some people don't like it. We may consider it.

I'm copy-pasting [a discussion we had with Sindre Sorhus](https://gist.github.com/sindresorhus/be6a363ea0bda1ce7e478785e35c6fb3#:~:text=talk%23194.-,I%20would%20have%20preferred%20the%20%22Find%20or%20Create%22%20and%20%22Quick%20Open,in%20the%20corner%20(we%20don%27t%20agree%20on%20too%20many%20subjective%20things).,-We%20think%20center.) about this:

> - I would have preferred the "Find or Create" and "Quick Open" to show up in the center. This is how most apps do it. Putting it at the bottom right means I have to look down all the time, which is annoying.

We've been experimenting with center and corner positioning for a lot of time. Here are some reasons we ended up with corner:
- We have a preview feature that shows the contents of the selected item in the editor. If the dialog is in the center, you will see less of the content.
- For dialogs that don't have the preview feature, we like it that you see more of your context. If your query is based on some text in the editor, you can look at the text and type in the dialog.
- That's subjective but we both (me and [@erusev](https://github.com/erusev)) like it in the corner (we don't agree on too many subjective things).

**Reply by vladstudio** [reply]
Thank you for detailed responses! All reasons you mentioned are perfectly valid - and it may be that I will grow to love it in the corner as well. Not critical anyway.

**Reply by vladstudio** [reply]
"some users sending us feedback thinking the app is native." - count me in! The app is so fast I thought it's native. Hmm!

**Reply by astoilkov** [reply]
Ok. Give it some time and write back here if you still think you want to be centered.
